### DISCUSSION TITLE
Untriaged list of useful features

### DISCUSSION THREAD

**Discussion by aurelienpierre** [aurelienpierreeng/ansel#60] [Ideas]
Draft
- [x] Force-recompute lighttable thumbnails

**Comment by lu-k** [comment]
Lighttable

- [ ] _Remove from library_ and _Delete from disk(trash)_ options in right click menu

**Comment by pedrorrodriguez** [comment]
- [ ] **Improved masks preview**: either black and white like Photoshop or grey for excluded areas like Davinci. The current yellow design implies selection, when it actually does the opposite. #769 
- [ ] **The image should not be recomputed while in mask preview mode**, only after leaving it. #749 
- [x] ~**Manual way to trigger a cascading white point normalization** for various scene referred modules from the lighttable.  For one or many images. Ideally the ability to include this in styles. #694~
- [ ] **Feedback during export** #771
- [ ] **Allow brightness to go below -25% in the primaries module** (manually) #753 
- [ ] **Drawn raster masks**. Basically your new drawing module, but not in a module and without color. Just to "paint" an opacity mask. The current drawn mask gets slow after 10-20 paths. If you don't care about editing the shape of the path after the fact, this should be faster, shouldn't it? This can be useful to "paint" with exposure, or do any number of things you usually do with masks.
- [ ] **Enable module presets and styles from search bar**
- [ ] #782 
- [ ] #781 
- [ ] #785 
- [ ] #789 
- [ ] #790
- [ ] #791
- [ ] #792
- [ ] #793
- [ ] #794
- [ ] #815

**Comment by pedrorrodriguez** [comment]
Older ideas from other users, moved over here from the issue tracker: 

- [x] #795 
- [ ] #796
- [ ] #797
